apps/mobile/src/libs/components/progress-bar/enums/enums.ts

```diff
@@ -0,0 +1 @@
+export { GradientCoordinates } from "./gradient-coordinates.enum";
```

Comment:
```suggestion
export { GradientCoordinate } from "./gradient-coordinates.enum";
```
B23. Don't forget about the file name